tests/testthat/test-build-home-index.R

```diff
@@ -160,3 +160,15 @@ test_that("allow email in BugReports", {
   html <- xml2::read_html(data_home_sidebar(pkg))
   expect_snapshot(xpath_xml(html, ".//li/a"))
 })
+
+test_that("ANSI are handled", {
+  withr::local_options(cli.num_colors = 256)
+  pkg <- local_pkgdown_site()
+
+  pkg <- pkg_add_file(pkg, "README.md", sprintf("prefer %s", cli::col_blue("a")))
```

Comment:
Minor nit: we wouldn't recommend this in `README.md`, only in `index.md` or even `pkgdown/index.md` ?

Thanks for taking this on!